plugins/generator-1.16.5/forge-1.16.5/triggers/command_executed.java.ftl

```diff
@@ -5,14 +5,22 @@
 			double i = entity.getPosX();
 			double j = entity.getPosY();
 			double k = entity.getPosZ();
-			String command=event.getParseResults().getReader().getString();
+			HashMap<String, String> command = new HashMap<>();
+            int index = -1;
+            for (String param : event.getParseResults().getReader().getString().split("\\s+")) {
+            	if (index >= 0)
+            		command.put(Integer.toString(index), param);
+            	index++;
+            }
+		    CommandContext<CommandSource> ctx = event.getParseResults().getContext().build(event.getParseResults().getReader().getString());
 			Map<String, Object> dependencies = new HashMap<>();
 			dependencies.put("x",i);
 			dependencies.put("y",j);
 			dependencies.put("z",k);
 			dependencies.put("world",entity.world);
 			dependencies.put("entity",entity);
-			dependencies.put("command",command);

```

Comment:
You did not add this back, which indicates you only added back command due to my request and did not test this... this makes PRs really hard to review as I need to be very careful with every detail due to it being hard to trust such PRs